CHANGELOG.md

```diff
@@ -1,12 +1,11 @@
 # drafter-npm CHANGELOG
 
-## Master
+## 2.0.0-pre.0
```

Comment:
Can you add mention that newer version of Drafter is used and link to the drafter release like as follows:

```
## 2.0.0-pre.0

This update now uses Drafter 4.0.0-pre.1. Please see [Drafter
4.0.0-pre.1](https://github.com/apiaryio/drafter/releases/tag/v4.0.0-pre.1) for
the list of changes.
```